cgi-bin/DW/Controller/Edges.pm

```diff
@@ -44,6 +44,9 @@ sub edges_handler {
         return $r->OK;
     };
 
+    # Don't allow access unless we have a logged-in user
+    return $error_out->( 404, 'Must log in first' ) unless LJ::get_remote();
```

Comment:
This is totally fine, I just am slightly weirded out by this function not invoking `controller(...)`, but maybe that's not super helpful or this kind of JSON thing? Weird tho, I thought all of our controllers called it... TIL.